crates/sui-types/src/sui_system_state/mod.rs

```diff
@@ -442,27 +442,35 @@ pub struct AdvanceEpochParams {
 
 #[cfg(msim)]
 pub mod advance_epoch_result_injection {
-    use crate::error::{ExecutionError, ExecutionErrorKind};
+    use crate::{
+        committee::EpochId,
+        error::{ExecutionError, ExecutionErrorKind},
+    };
     use std::cell::RefCell;
 
     thread_local! {
-        static OVERRIDE: RefCell<bool>  = RefCell::new(false);
+        static OVERRIDE: RefCell<Option<(EpochId, EpochId)>>  = RefCell::new(None);
```

Comment:
Why 2 ids here? Could you doc them?

Reply:
It's in the doc for `set_override`. Will add comments here too